src/block/expand/edit.js

```diff
@@ -58,7 +58,7 @@ const TEMPLATE = [
 		text: __( 'Show less', i18n ),
 		linkUrl: '#',
 		className: 'is-style-link stk-block-expand__hide-button',
-		customAttributes: [ [ 'aria-hidden', 'true' ], [ 'role', 'button' ] ],
+		customAttributes: [ [ 'aria-hidden', 'true' ], [ 'role', 'button' ], [ 'aria-expanded', 'false' ] ],
```

Comment:
Just like the above, this should be `true`
```suggestion
		customAttributes: [ [ 'aria-hidden', 'true' ], [ 'role', 'button' ], [ 'aria-expanded', 'true' ] ],
```